docs/sources/panels-visualizations/visualizations/traces/index.md

```diff
@@ -13,16 +13,89 @@ weight: 850
+- [Tempo data source]({{< relref "../../../datasources/tempo/" >}})
 - [Getting started with Tempo](/docs/tempo/latest/getting-started)
 
 {{< figure src="/static/img/docs/explore/trace-view-9-4.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}
+
+## Add a panel with tracing visualizations
+
+Once you have tracing data available in your Grafana instance, you can add tracing panels to your Grafana dashboards.
+
+Using a dashboard variable, `traceID`, lets you create a query to show specific traces for a given trace ID.
+For more information about dashboard variables, see the [Variables documentation]({{< relref "../../../dashboards/variables" >}}).
+
+### Before you begin
+
+To use this procedure, you need:
+
+- A Grafana instance
+- A Tempo data source connected to your Grafana instance (see [documentation](/docs/grafana/datasources/tempo/))
+
+### Add the Tracing panel query
+
+You need to add the tracing panel to your dashboard and define a query using the Panel editor.
+The query determines the data that is displayed in the panel.
+For more information on the Panel editor, see the [Panel editor]({{< relref "../../panel-editor-overview" >}}) documentation.
+
+This procedure uses dashboard variables and templates to allow us to enter trace IDs which can then be visualized. We’re going to use a variable called `traceId` and add it as a template query.
+
+1. From your Grafana instance, create a new dashboard or go to an existing dashboard where you would like to add tracing panels.
+1. Select **Add visualization** from a new dashboard or select **Add Panel** on an existing dashboard.
+1. Search for and select the appropriate tracing data source.
+1. In the panel search pane on the right, select the **Visualizations** tab, search for and select for Traces.
+1. On the **Panel editor**, enter a **Title** for your trace panel. For more information on the panel editor, see the [Panel editor documentation]({{< relref "../../panel-editor-overview" >}}).
+1. Select the **TraceQL** query type tab from the query editor.
+1. Enter `${traceId}` in the TraceQL query field to create a dashboard variable. This variable is used as the template query. <br /> {{< figure src="/static/img/docs/panels/traces/screenshot-traces-template-query.png" caption="Add a template query" >}}
+
+1. Select **Apply** in the Panel editor to add the panel to the dashboard.
+1. Expand the size of the trace panel so that it fills the width of the dashboard. We’ll now use the dashboard settings to create the variable used in the panel’s template query.
+1. Go to the dashboard **Settings** and add a new variable called `traceId`, of variable type **Custom**, giving it a label if required. Select **Apply** to add the variable to the dashboard. <br /> {{< figure  src="/static/img/docs/panels/traces/screenshot-traces-custom-variable.png" maxWidth="25%" caption="Add a Custom variable" >}}
```

Comment:
Agreed. I fought with this image last night. The reason I used the figure shortcode was because it offers a maxWidth capability. Or at least it's supposed to. Unfortunately, no matter what I did, I couldn't control the size of the image. I could change to HTML mark up instead. If all else fails, I can resize the image and upload it again.

Reply:
Fixed. I should have used max-width instead of maxWidth. I will update the writers-toolkit.